advanced/javascript/make-http-s-requests.md

```diff
@@ -85,3 +85,59 @@ const response = http.request('https://example.com`, {
 | `ok`       | `true` if request was successful, `false` otherwise |
 | `status`   | HTTP status code (i.e. `200`)                       |
 | `body`     | String body of the response                         |
+
+### Example
```

Comment:
If you feel this would be better placed in the examples section, I'm open to that. Those seemed like much more comprehensive/complete examples, whereas the idea here was just that the JS syntax allowed is fairly particular, successfully chaining requests can be a little tricky, and an example at-hand might be useful, particularly for people who don't think to go to the examples (I didn't, until Manish Varma directed me to them in the Maestro Slack.)